docs/subnets/create-first-subnet.md

```diff
@@ -0,0 +1,208 @@
++-------+----------+------------------------------------------------------------------------------------+
+| node2 | mySubnet | http://127.0.0.1:9652/ext/bc/SPqou41AALqxDquEycNYuTJmRvZYbfoV9DYApDJVXKXuwVFPz/rpc |
++-------+----------+------------------------------------------------------------------------------------+
+| node3 | mySubnet | http://127.0.0.1:9654/ext/bc/SPqou41AALqxDquEycNYuTJmRvZYbfoV9DYApDJVXKXuwVFPz/rpc |
++-------+----------+------------------------------------------------------------------------------------+
+| node4 | mySubnet | http://127.0.0.1:9656/ext/bc/SPqou41AALqxDquEycNYuTJmRvZYbfoV9DYApDJVXKXuwVFPz/rpc |
++-------+----------+------------------------------------------------------------------------------------+
+| node5 | mySubnet | http://127.0.0.1:9658/ext/bc/SPqou41AALqxDquEycNYuTJmRvZYbfoV9DYApDJVXKXuwVFPz/rpc |
++-------+----------+------------------------------------------------------------------------------------+
+| node1 | mySubnet | http://127.0.0.1:9650/ext/bc/SPqou41AALqxDquEycNYuTJmRvZYbfoV9DYApDJVXKXuwVFPz/rpc |
++-------+----------+------------------------------------------------------------------------------------+
+
+Browser Extension connection details (any node URL from above works):
+RPC URL:          http://127.0.0.1:9650/ext/bc/SPqou41AALqxDquEycNYuTJmRvZYbfoV9DYApDJVXKXuwVFPz/rpc
+Funded address:   0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC with 1000000 (10^18) - private key: 56289e99c94b6912bfc12adc093c9b51124f0dc54ac7a766b2bc5ccf558d8027
+Network name:     mySubnet
+Chain ID:         54325
+Currency Symbol:  TUTORIAL
+```
+
+You can use the deployment details to connect to and interact with your subnet. Now let's interact with it.
+
+## Interacting with Your Subnet
+
+You can use the value provided by `Browser Extension connection details` to connect to your Subnet with Metamask, Core, or any other wallet.
+
+```text
+Browser Extension connection details (any node URL from above works):
+RPC URL:          http://127.0.0.1:9650/ext/bc/SPqou41AALqxDquEycNYuTJmRvZYbfoV9DYApDJVXKXuwVFPz/rpc
+Funded address:   0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC with 1000000 (10^18) - private key: 56289e99c94b6912bfc12adc093c9b51124f0dc54ac7a766b2bc5ccf558d8027
+Network name:     mySubnet
+Chain ID:         54325
+Currency Symbol:  TUTORIAL
+```
+
+In this tutorial, we'll use Metamask.
+
+### Importing the Test Private Key
+
+:::warning
+This address's private key is well-known. Anyone can steal funds sent to this address. Only use it on development networks that only you have access to. If production funds are sent to this address, they will likely be stolen instantly.
+:::
+
+First, you need to import your airdrop private key into Metamask. Do this by clicking the profile bubble in the top right corner and select `Import account`.
+
+![Import Metamask Account](/img/metamask-import.png)
+
+Import the well-known private key `0x56289e99c94b6912bfc12adc093c9b51124f0dc54ac7a766b2bc5ccf558d8027`.
```

Comment:
I know you are saying it 100 times and making it clear. However, I wonder if we could just get by without this importing and have the user create a private key instead or something. Just trying to avoid issues....

Reply:
I mean, if we do that, then I think the place to remove it is from the creation menu so we don't use test keys anywhere and it's always bring-your-own-keys. At the end of the day, this is how to interact with your subnet. I think we can trust anyone who makes it this far to be ok.